src/Screens/ConfirmationScreen/confirmationPage.tsx

```diff
@@ -1,10 +1,233 @@
+                    </Text>
+                  </View>
+                ) : null}
+                <View style={confirmationPageStyles.priceAndIconsContainer}>
+                  <Text style={confirmationPageStyles.priceItem}>
+                    {item.dish.price} BYN
+                  </Text>
+                  <Text style={confirmationPageStyles.priceItem2}>
+                    за 1 позицию
+                  </Text>
+                </View>
+              </View>
+            </View>
+          ))}
+          <View style={confirmationPageStyles.lineView}></View>
+          <View style={confirmationPageStyles.itemsWrapperContainer}>
+            <View style={confirmationPageStyles.itemWrapperStyle}>
+              <Text style={confirmationPageStyles.firstPartTextStyle}>
+                Тип заказа:
+              </Text>
+              <Text style={confirmationPageStyles.secondPartTextStyle}>
+                {orderData.orderType}
+              </Text>
+            </View>
+            {orderData.adress ? (
+              <View style={confirmationPageStyles.itemWrapperStyle}>
+                <Text style={confirmationPageStyles.firstPartTextStyle}>
+                  Адрес:
+                </Text>
+                <Text
+                  style={confirmationPageStyles.secondPartTextStyle}
+                  key={uuid.v4()}
+                >
+                  {formatAdress(orderData.adress)}
+                </Text>
+              </View>
+            ) : null}
+            <View style={confirmationPageStyles.itemWrapperStyle}>
+              <Text style={confirmationPageStyles.firstPartTextStyle}>
+                Дата:
+              </Text>
+              <Text style={confirmationPageStyles.secondPartTextStyle}>
+                {orderData.date}
+              </Text>
+            </View>
+            <View style={confirmationPageStyles.itemWrapperStyle}>
+              <Text style={confirmationPageStyles.firstPartTextStyle}>
+                Время:
+              </Text>
+              <Text style={confirmationPageStyles.secondPartTextStyle}>
+                {orderData.time}
+              </Text>
+            </View>
+            <View style={confirmationPageStyles.contactInformationStyleWrapper}>
+              <Text style={confirmationPageStyles.firstPartTextContactStyle}>
+                Контактная информация:
+              </Text>
+
+              <Text style={confirmationPageStyles.secondPartTextContactStyle}>
+                Кристина, +375296457721
+              </Text>
+            </View>
+            {orderData.table ? (
+              <View style={confirmationPageStyles.itemWrapperStyle}>
+                <Text style={confirmationPageStyles.firstPartTextStyle}>
+                  Стол на
+                </Text>
+                <Text style={confirmationPageStyles.secondPartTextStyle}>
+                  {orderData.table}
+                </Text>
+              </View>
```

Comment:
move to the common component and reuse after

Reply:
fixed